react-field-agent-assessment/src/Components/Popup.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import("./AgentList.css");
+
+function Popup({ handleDeleteTrue, handleDeleteFalse }) {
+  return (
+    <div className="popup">
+      <div className="popup-contents">
+        <p>Are you sure you want to delete? </p>
+        <button onClick={handleDeleteFalse} className="popup-cancel">
+          Cancel
+        </button>
+        <button onClick={handleDeleteTrue} className="popup-confirm">
+          Confirm
+        </button>
+      </div>
+    </div>
```

Comment:
Opportunity for Improvement: Your pop-up looks good and the delete/cancel functions work. the requirements ask for display of the basic agent data. Since all of your visual elements are on the same screen the agent name is visible. A more polished look would be to display that info in the pop-up along with the delete confirmation request.